src/components/fees/HistoricalStakersDistributionList.tsx

```diff
@@ -0,0 +1,119 @@
+import { useMemo } from 'react'
+import styled from 'styled-components'
+
+import { getAddress } from '@ethersproject/address'
```

Comment:
@0xDmitry I added this because the addresses coming from the subgraph are all in lowercase which means
`item.address === userAddress` always returns false.

Fix: `getAddress(item.address) === userAddress`

Reply:
Good catch, thx!